checks/check_extra7185

```diff
@@ -0,0 +1,85 @@
+
+    # Use --scope Local to list only Customer Managed Policies
+    # Query 'Policies[?AttachmentCount > `0`]'  to check if this policy is in use, so attached to any user, group or role
+    LIST_CUSTOM_POLICIES=$(${AWSCLI} iam list-policies ${PROFILE_OPT} \
+    --scope Local \
+    --query 'Policies[*].[Arn,DefaultVersionId]' \
+    --output text)
+
+    # Check errors
+    if grep -q -E 'AccessDenied|UnauthorizedOperation|AuthorizationError' <<< "${LIST_CUSTOM_POLICIES}"; then
+        textInfo "${REGION}: Access Denied trying to list IAM policies" "${REGION}"
+    else
+        if [[ $LIST_CUSTOM_POLICIES ]]; then
+            while read -r POLICY_ARN POLICY_DEFAULT_VERSION; do
+                POLICY_PRIVILEGED_ACTIONS=$($AWSCLI iam get-policy-version ${PROFILE_OPT} \
+                    --policy-arn "${POLICY_ARN}" \
+                    --version-id "${POLICY_DEFAULT_VERSION}" \
+                    --query "PolicyVersion.Document.Statement[]" \
+                    --output json)
+
+                if grep -q -E 'AccessDenied|UnauthorizedOperation|AuthorizationError' <<< "${POLICY_PRIVILEGED_ACTIONS}"; then
+                    textInfo "${REGION}: Access Denied trying to get policy version" "${REGION}"
+                    continue
+                fi
+
+                ALLOWED_ACTIONS=$(jq -r '.[] | select(."Effect" == "Allow") | .Action // empty' <<< "${POLICY_PRIVILEGED_ACTIONS}" | sed 's/\[//;s/\]//;s/,/ /;s/ //g;/^$/d')
+                DENIED_ACTIONS=$(jq -r '.[] | select(."Effect" == "Deny") | .Action // empty' <<< "${POLICY_PRIVILEGED_ACTIONS}" | sed 's/\[//;s/\]//;s/,/ /;s/ //g;/^$/d')
+                DENIED_NOT_ACTIONS=$(jq -r '.[] | select(."Effect" == "Deny") | .NotAction // empty' <<< "${POLICY_PRIVILEGED_ACTIONS}" | sed 's/\[//;s/\]//;s/,/ /;s/ //g;/^$/d')
+
+                # First, we need to perform a left join with ALLOWED_ACTIONS and DENIED_ACTIONS
+                LEFT_ACTIONS=$(diff <(echo "${ALLOWED_ACTIONS}") <(echo "${DENIED_ACTIONS}") | grep "^<" | sed 's/< //;s/"//g')
+                # Then, we need to find the DENIED_NOT_ACTIONS in LEFT_ACTIONS
+                PRIVILEGED_ACTIONS=$(comm -1 -2  <(sort <<< "${DENIED_NOT_ACTIONS}") <(sort <<< "${LEFT_ACTIONS}"))
+                # Finally, check if there is a privilege escalation action within this policy
+                POLICY_PRIVILEGE_ESCALATION_ACTIONS=$(grep -o -E "${PRIVILEGE_ESCALATION_IAM_ACTIONS}" <<< "${PRIVILEGED_ACTIONS}")
+                if [[ -n "${POLICY_PRIVILEGE_ESCALATION_ACTIONS}" ]]; then
+                    textFail "${REGION}: Customer Managed Policy ${POLICY_ARN} allows for privilege escalation using the following actions: ${POLICY_PRIVILEGE_ESCALATION_ACTIONS//$'\n'/ }" "${REGION}" "${POLICY_NAME}"
```

Comment:
```suggestion
                    textFail "${REGION}: Custom IAM policy ${POLICY_ARN} allows for privilege escalation using the following actions: ${POLICY_PRIVILEGE_ESCALATION_ACTIONS//$'\n'/ }" "${REGION}" "${POLICY_NAME}"
```

Reply:
Why not Customer Managed? Is the way AWS calls this kind of policies.

![Screenshot 2022-05-30 at 16 59 32](https://user-images.githubusercontent.com/16007882/171018653-1702d952-a498-4b87-84de-57e715f380ff.png)